### ISSUE TITLE
build/readme is incomplete

### ISSUE THREAD

**Issue by ohenley** [Roldak/Madarch#1]
The project looks super nice but it is a bummer to have to fetch / install dependencies by hand.
It would be nice to build using `alr` or have the repo synched on working submodules snapshots or a heads-up in the readme.

thx!

**Comment by Roldak** [comment]
Hi Olivier!

> The project looks super nice

Thanks! :smiley: 

> it is a bummer to have to fetch / install dependencies by hand.
It would be nice to build using alr or have the repo synched on working submodules snapshots or a heads-up in the readme.

You're right. I tried using Alire when I started this project but I had some problems, I think the OpenGLAda version was too old or not getting along with the GNAT Community build at the time.

But I just re-tried with an Alire-only setup and it seems to work just fine now, even for getting the toolchain, which is really great.

I'll try to update the setup to work with Alire when I get the chance to. I also noticed I had some work locally to add volumetric lights, I'll try push that as well and fix all the examples!
